calendar-bundle/src/Resources/contao/classes/Calendar.php

```diff
@@ -144,20 +144,18 @@ protected function generateFiles($arrFeed)
 					continue;
 				}
 
+				$objParent = PageModel::findWithDetails($jumpTo);
```

Comment:
looks like this is bypassing the `$arrUrls` cache?

Reply:
Yes, setting the global `$objPage` variable is unrelated to the URL cache and needs to be done for each record. We could implement a separate page cache, since `findWithDetails` cannot use the model registry (for the object with loaded details).

Reply:
pretty sure that will be a big performance penalty 😞

Reply:
I am not sure about this, either. @contao/developers WDYT?

Reply:
The performance penalty should be rectified now with the implemented static cache.

Reply:
I agree that should be okay.

Reply:
Ok. 😊